packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -118,6 +150,42 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
     return instance;
   }
 
+  /**
+   * Checks and creates an authorization module based on the provided authorization module type.
+   *
+   * @param {Signer} signer - The signer object used for module creation.
+   * @param {AuthorizationModuleType} authorizationModuleType - The type of authorization module to create.
+   * @returns {Promise<BaseValidationModule>} A promise that resolves to the created authorization module.
+   */
+  async checkAndCreateModule(signer: Signer, authorizationModuleType: AuthorizationModuleType): Promise<BaseValidationModule> {
+    switch (authorizationModuleType) {
+      case AuthorizationModuleType.ECDSA_OWNERSHIP:
+        return ECDSAOwnershipValidationModule.create({
+          signer,
+          moduleAddress: authorizationModuleType,
+        });
+      case AuthorizationModuleType.MULTICHAIN:
+        return MultiChainValidationModule.create({
+          signer,
+        });
+      case AuthorizationModuleType.SESSION:
```

Comment:
session key manager module or batched session router module can not be default authorization modules.